public/data/nikhil96.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Nikhil Sharma",
+  "type": "personal",
+  "bio": "Learning to contribute!",
+  "avatar": "https://github.com/itsmenikhill.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/itsmenikhill",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/itsnikhilpeeps",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/itsnikhilhere",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Working Full Time",
+      "date": "January 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started Working Full Time",
+      "url": "https://www.eddiejaoude.io/itsmenikhill"
```

Comment:
I believe you need to update this link.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix